src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -682,6 +682,13 @@ void DoorShutter_Draw(Actor* thisx, GlobalContext* globalCtx) {
         s32 pad[2];
         ShutterInfo* sp70 = &D_80998134[this->unk_16C];
 
+        //! @bug In very niche scenarios, depending on the placement of other doors when a new room is loading,
+        //! it is possible that a newly spawned door will try to draw after init, without having run it's first update.
+        //! In this case, `DoorShutter_SetupDoor` will not have run yet, so unk_16C is not initialized.
+        //! When this occurs, it will try to draw the display list of the first entry in `D_80998134`.
+        //! This bug can be avoided either by initialized 16C in init, or by checking 
```

Comment:
```suggestion
        //! This bug can be avoided either by initialized unk_16C in init, or by checking 
```
Recommend using the full var name so a find/replace will catch it easily when it's renamed.